### DISCUSSION TITLE
listen unix $SEMAPHORE_TMP_PATH/ssh-agent-64-ef4sqvntep.sock: bind: operation not supported

### DISCUSSION THREAD

**Discussion by mobiservice19** [semaphoreui/semaphore#2355] [General]
Hi, I get the following error when starting any task:

Semaphore v2.10.22-e44910d-1721658561

```bash
Failed updating repository: listening on socket "/tmp/semaphore/ssh-agent-64-ef4sqvntep.sock": listen unix /tmp/semaphore/ssh-agent-64-ef4sqvntep.sock: bind: operation not supported

Running app failed: listening on socket "/tmp/semaphore/ssh-agent-64-ef4sqvntep.sock": listen unix /tmp/semaphore/ssh-agent-64-ef4sqvntep.sock: bind: operation not supported
```

**Comment by fiftin** [comment]
Hi @mobiservice19 what installation do you use (docker or binary or snap)?

**Reply by mobiservice19** [reply]
I use Kubernetes.
When using docker, the result is the same

**Reply by fiftin** [reply]
Do you use official image? What version of Semaphore?

**Reply by mobiservice19** [reply]
image: semaphoreui/semaphore:v2.10.22

**Comment by fiftin** [comment]
![image](https://github.com/user-attachments/assets/e137b03e-ccfc-4a08-9dfa-53f3d555a7a3)

Is it your case?
